File: uikit/src/main/java/utils/AudioVisualizer/AudioBarConstant.java
package utils.AudioVisualizer;

import android.graphics.Color;

public class AudioBarConstant {

    public static enum AnimSpeed {
        SLOW,
        MEDIUM,
        FAST
    }
    public enum PaintStyle {
        OUTLINE,
        FILL
    }
    public enum PositionGravity {
        TOP,
        BOTTOM
    }
    public static class AudioBarConstants {
        public static final float DEFAULT_DENSITY = 0.25f;
        public static final int DEFAULT_COLOR = Color.BLACK;
        public static final float DEFAULT_STROKE_WIDTH = 6.0f;
        public static final int MAX_ANIM_BATCH_COUNT = 4;
    }
}

File: uikit-kotlin/src/main/java/com/cometchat/pro/uikit/AudioVisualizer/AudioBarConstant.kt
package com.cometchat.pro.uikit.AudioVisualizer

import android.graphics.Color

class AudioBarConstant {
    enum class AnimSpeed {
        SLOW, MEDIUM, FAST
    }

    enum class PaintStyle {
        OUTLINE, FILL
    }

    enum class PositionGravity {
        TOP, BOTTOM
    }

    object AudioBarConstants {
        const val DEFAULT_DENSITY = 0.25f
        const val DEFAULT_COLOR = Color.BLACK
        const val DEFAULT_STROKE_WIDTH = 6.0f
        const val MAX_ANIM_BATCH_COUNT = 4
    }
}